src/nvim/lua/executor.c

```diff
@@ -613,6 +614,9 @@ static void nlua_common_vim_init(lua_State *lstate, bool is_thread, bool is_stan
   lua_pushvalue(lstate, -3);
   lua_setfield(lstate, -2, "luv");
   lua_pop(lstate, 3);
+
+  // internal vim._fs... API
+  nlua_add_fs(lstate);
```

Comment:
the other calls above are inlined, do we need the separate `nlua_add_fs` function?

Reply:
Not really, `nlua_add_fs` was created when the call was in `nlua_state_init`. Now that it's in `nlua_common_vim_init`, it's better to inline it